src/components/Testimonial.astro

```diff
@@ -0,0 +1,26 @@
+---
+    export interface Props {
+        mainImageAlt?: string;
+        quote: string;
+        authorInformation: string;
+        authorDescription: string;
+        page:string;
+    }
+
+    const { mainImageAlt = " ", quote, authorInformation, authorDescription, page } = Astro.props;
+---
+
+<section class={`testimonial testimonial--${page}`}>
+    <picture class="testimonial__shape-container">
+        <source media="(min-width: 1024px)" srcset={`/assets/images/${page}/testimonial/testimonial-shape-desktop.svg`}>
+        <img class="testimonial__shape" src={`/assets/images/${page}/testimonial/testimonial-shape-mobile.svg`} alt=" ">
```

Comment:
I think this would be better if instead of sending a Prop "page" maybe we can send shapeType or something like that this way we can just send "top" or "bottom" even better set the defualt to top and make the shapeType Prop optional. I would also move all shapeTypes vectors to the "vectors" folder and create a shapes subfolder so we can have them all live in the same place just different formats/styles.
```suggestion
        <img class="testimonial__shape" src={`/assets/images/${shapeType}/testimonial/testimonial-shape-mobile.svg`} alt=" ">

        Even better IMO
        <img class="testimonial__shape" src={`/assets/images/testimonial-shapes/${shapeType}/testimonial-shape-mobile.svg`} alt=" ">
```